internal/client.go

```diff
@@ -1288,3 +1296,17 @@ func SetCallbacksOnStartWorkflowOptions(opts *StartWorkflowOptions, callbacks []
 func SetLinksOnStartWorkflowOptions(opts *StartWorkflowOptions, links []*commonpb.Link) {
 	opts.links = links
 }
+
+// SetOnConflictOptionsOnStartWorkflowOptions is an internal only method for setting conflict
+// options on StartWorkflowOptions.
+// OnConflictOptions are purposefully not exposed to users for the time being.
+func SetOnConflictOptionsOnStartWorkflowOptions(opts *StartWorkflowOptions) {
+	if opts.WorkflowIDConflictPolicy == enumspb.WORKFLOW_ID_CONFLICT_POLICY_UNSPECIFIED {
+		opts.WorkflowIDConflictPolicy = enumspb.WORKFLOW_ID_CONFLICT_POLICY_USE_EXISTING
+	}
```

Comment:
Not convinced we want to change the default conflict policy, especially not in a way that's hard to discover.
Without reading this function's implementation, I'd never guess that the conflict policy changes implicitly here.
Let's revert this change to the default behavior and let the user opt into this.

Reply:
Sure, I thought we wanted to make Nexus requests to always use this conflict policy.